cluster-autoscaler/cloudprovider/gce/mig_info_provider.go

```diff
@@ -245,6 +249,13 @@ func (c *cachingMigInfoProvider) isMigCreatingOrDeletingInstances(mig Mig) bool
 
 // updateMigInstancesCache updates the mig instances for each mig
 func (c *cachingMigInfoProvider) updateMigInstancesCache(migToInstances map[GceRef][]GceInstance) error {
+	listingStartTime := time.Now()
+	defer func() { metrics.UpdateDuration(metrics.BulkListMigInstances, time.Since(listingStartTime)) }()
```

Comment:
nit: ditto, you can use `UpdateDurationFromStart` and there's no need to wrap into `func()`.

Reply:
Updated